src/components/PopupWithConfirmation.js

```diff
@@ -30,10 +30,5 @@ export default class PopupWithConfirmation extends Popup {
   // Override the setEventListeners to set default popup behaviors
   setEventListeners() {
     super.setEventListeners(); // Inherit the default listeners
```

Comment:
You shouldn’t create the **same methods** of the parent class if you don’t change them inside and only call them with `super`. The parent methods are available in any inheritor of the class.